app/assets/javascripts/common/my_facilities.js

```diff
@@ -54,7 +54,6 @@ function createBaseTrendChartConfig() {
           radius: 0,
         },
       },
-      responsive: true,
```

Comment:
Responsive default values = true 

This is not required
https://www.chartjs.org/docs/3.9.1/configuration/responsive.html